cmd/cape/cmd/function.go

```diff
@@ -0,0 +1,87 @@
+package cmd
+
+import (
+	"fmt"
+	"os"
+	"path/filepath"
+
+	"github.com/spf13/cobra"
+)
+
+var functionCmd = &cobra.Command{
+	Use:   "function",
```

Comment:
Looks good. I do wonder if this should simply be `cape create`

Reply:
Hmmmm I think we will have another command under `function`. The one to build dependencies.

```
cape function create ....
cape function build ....
```

Reply:
But both could just be under `cape` directly.

Reply:
I do think it's nice when you can look at the top level and see the most important stuff.

I feel like build could be part of deploy. Like `cape deploy --build` . Unless we see a reason to build separately from deploy 

Some user feedback has suggested that `cape test` is confusing and would be better as a flag to run.

Reply:
Good point. I like `cape deploy --build`.

Reply:
I just moved it to top level for now. We can always move things around with top level becomes too cluttered.